acl/src/main/java/org/apache/rocketmq/acl/plain/PlainAccessResource.java

```diff
@@ -268,7 +269,9 @@ public static PlainAccessResource parse(GeneratedMessageV3 messageV3, Authentica
                 }
             } else if (NotifyClientTerminationRequest.getDescriptor().getFullName().equals(rpcFullName)) {
                 NotifyClientTerminationRequest request = (NotifyClientTerminationRequest) messageV3;
-                accessResource.addGroupResourceAndPerm(request.getGroup(), Permission.SUB);
+                if (StringUtils.isNotBlank(request.getGroup().getName())) {
```

Comment:
Can you consider replacing `StringUtils.isNotBlank` with `org.apache.rocketmq.common.UtilAll#isBlank`.

Reply:
It is the same. the UtilAll#isBlank is using StringUtils.isNotBlank